configs/config.json.example

```diff
@@ -7,23 +7,35 @@
     "libencrypt_location": "",
     "tasks": [
       {
-        "type": "HandleSoftBan"
+        "type": "HandleSoftBan",
+        "config": {
```

Comment:
I don't think we should add these to the examples. It is clutter and people will think it is required. Maybe add some other task and say enabled false, just to give an example.